make/harbor.cfg

```diff
@@ -133,8 +142,8 @@ clair_db_username = postgres
 
 #Clair default database
 clair_db = postgres
-################### end of HA section #####################
 
+##########End of Clair DB configuration############
 #************************END INITIAL PROPERTIES************************
```

Comment:
We should only put the properties that take effect in the first boot into the section between `BEGIN INITIAL PROPERTIES` and `END INITIAL PROPERTIES`.